www/js/app.js

```diff
@@ -1,10 +1,6 @@
 Stripe.setPublishableKey("pk_test_WwMFp1CE94C8P8QLtPrzW5Lq"); 
 
-var app = angular.module('starter', 
-    ['ionic', 
-     'ngCordova', 
-     'ion-datetime-picker', 
-     'credit-cards'])
+var app = angular.module('starter', ['ionic', 'ngCordova', 'ion-datetime-picker', 'credit-cards'])
```

Comment:
Please leave code style as it was. Lists of more than three items should be on separate lines for readability.